t/06-blacklist.t

```diff
@@ -1,10 +1,13 @@
 #!/usr/bin/perl
 
+use LedgerSMB::Sysconfig;
```

Comment:
Using File::Temp would mean a new parameter for tools/makeblacklist.pl, which would mean parsing, the command line, validating...
The objective of the PR is to allow a read-only source tree. Proper usage of temp files and removal of sysconfig dependency should be the object of another, which address that more globally.